App.js

```diff
@@ -1,26 +1,47 @@
-import React from 'react'
-import styled from 'styled-components/native'
+import React from "react";
+import { NavigationContainer } from "@react-navigation/native";
+import { createStackNavigator } from "@react-navigation/stack";
 
-const Container = styled.View`
-  flex: 1;
-  background-color: papayawhip;
-  justify-content: center;
-  align-items: center;
-`
+import HomeScreen from "./components/HomeScreen";
+import ListScreen from "./components/ListScreen";
+import Rembrandt from "./components/Rembrandt";
+import Vermeer from "./components/Vermeer";
+import ArtDetails from "./components/ArtDetails";
 
-const Title = styled.Text`
-  font-size: 24px;
-  color: palevioletred;
-`
+const Stack = createStackNavigator();
 
 const App = () => {
   return (
-    <Container>
-      <Title>This is your cool app!</Title>
-      <Title>Go to App.js and start coding</Title>
-      <Title>💅💅💅</Title>
-    </Container>
-  )
-}
+    <NavigationContainer>
+      <Stack.Navigator initialRouteName="Home">
+        <Stack.Screen
+          name="Home"
+          component={HomeScreen}
+          options={{ title: "Explore" }}
+        />
```

Comment:
you repeated this code a few times. Remember to use .map() to generate repeatable code in the future 🤓